test/reentrancy/AttackAccount.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.0;
+
+import { console } from "forge-std/Test.sol";
+import { TokenController } from "../../contracts/TokenController.sol";
+import { MiniMeToken } from "../../contracts/MiniMeToken.sol";
+
+contract AttackAccount {
+    address public owner = msg.sender;
+    MiniMeToken public token;
+
+    constructor(MiniMeToken _token) {
+        token = _token;
+    }
+
+    function attack(address _from, address _to, uint256 _amount) external {
+        token.transferFrom(_from, _to, _amount);
+    }
```

Comment:
Would be cool if we could remove `console` and `TokenController` imports here as they are indeed not needed.